cfme/tests/ansible/test_embedded_ansible_basic.py

```diff
@@ -214,6 +215,49 @@ def test_embedded_ansible_repository_invalid_url_crud(request, appliance, wait_f
     repository.delete_if_exists()
 
 
+@pytest.mark.rhel_testing
+@pytest.mark.tier(1)
+def test_embedded_ansible_add_private_repository_crud(
+    request, credentials_collection, appliance, wait_for_ansible
+):
+    """
+    Add Private Repository by using SCM credentials.
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: Ansible
+        caseimportance: critical
+        initialEstimate: 1/6h
+        tags: ansible_embed
+    """
+    creds = {
+        "username": conf.credentials["scm_creds"]["username"],
+        "password": conf.credentials["scm_creds"]["username"],
+    }
+
+    scm_credential = credentials_collection.create(
+        "Scm_credential",
+        "Scm",
+        **creds
+    )
+    repositories = appliance.collections.ansible_repositories
+    repository = repositories.create(
+        name=fauxfactory.gen_alpha(),
+        url="https://gitlab.com/cfme/ansible_private_repo.git",
+        description=fauxfactory.gen_alpha(),
+        scm_credentials=scm_credential.name,
+    )
+    view = navigate_to(repository, "Details")
+    assert (
+        view.entities.summary("Properties").get_text_of("Status") == "failed"
+        if appliance.version < "5.11"
+        else "error"
+    )
+
+    scm_credential.delete_if_exists()
```

Comment:
Please delete these in finalizer method before assert 
 
     @request.addfinalizer
     def _cleanup():
         scm_credential.delete_if_exists()
         repository.delete_if_exists()

Reply:
@sshveta I intentionally not added finalizer, because of CRUD tests.

Reply:
Adding Finalizer.